src/definitions.ts

```diff
@@ -37,6 +37,25 @@ export interface MediaFetchOptions {
    * Which album identifier to query in (get identifier with getAlbums())
    */
   albumIdentifier?: string;
+  /**
+   * Sort order of returned assets by field and ascending/descending
+   */
+  sort?: string | MediaSort[];
+}
+
+export interface MediaSort {
+  key:
```

Comment:
I think instead of string here we could extract this as a separate type (eg: `Descriptor` maybe?) 

```
sort?: Descriptor | MediaSort[]
```